pkg/metadata/inventories_collector.go

```diff
@@ -50,11 +51,29 @@ func (c inventoriesCollector) Send(ctx context.Context, s serializer.MetricSeria
 	return nil
 }
 
+// getMinInterval gets the inventories_min_interval value, applying the default if it is zero.
+func getMinInterval() time.Duration {
+	minInterval := time.Duration(Datadog.GetInt("inventories_min_interval")) * time.Second
+	if minInterval <= 0 {
+		minInterval = DefaultInventoriesMinInterval * time.Second
+	}
+	return minInterval
+}
+
+// getMaxInterval gets the inventories_max_interval value, applying the default if it is zero.
+func getMaxInterval() time.Duration {
+	maxInterval := time.Duration(Datadog.GetInt("inventories_max_interval")) * time.Second
```

Comment:
The build errors are not wrong here - we're using a variable not defined in this file.